packages/react/src/types/component-props.ts

```diff
@@ -481,15 +596,18 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
    */
   fitViewOptions?: FitViewOptions;
   /**
-   *The connectOnClick option lets you click or tap on a source handle to start a connection
+   * The `connectOnClick` option lets you click or tap on a source handle to start a connection
    * and then click on a target handle to complete the connection.
    *
-   * If you set this option to false, users will need to drag the connection line to the target
+   * If you set this option to `false`, users will need to drag the connection line to the target
    * handle to create a connection.
+   * @default true
```

Comment:
https://github.com/xyflow/xyflow/blob/b9e4f829339c2e3dd489dba2f207e11fb7c67f7c/packages/react/src/store/initialState.ts#L117

is it right?